buildenv/jenkins/testJobTemplate

```diff
@@ -225,9 +225,12 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('OPENJDK_SHA', "", "OpenJDK SHA from which to run tests")
 							stringParam('TKG_OWNER_BRANCH', "AdoptOpenJDK:master", "TestKitGen git [owner]:[branch]. Default is AdoptOpenJDK:master")
 							stringParam('TKG_SHA', "", "TKG sha")
+              stringParam('ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH', "Personal Adoptopenjdk/openjdk-systemtest repository owner:branch", "AdoptOpenJDK:master")
```

Comment:
2 of these repositories now live in a new github organization, so I think we can update them to this:

ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH, "Personal Adoptium/aqa-systemtest repository owner:branch", Adoptium:master

STF_OWNER_BRANCH, "Personal stf repo owner:branch", Adoptium:master

They have been moved from AdoptOpenJDK to Adoptium github org, so if we can switch each of these comments and defaults to use Adoptium that will be preferable.

Reviewers will then run a Grinder test to verify this will work.